lib/pages/character/inventory.dart

```diff
@@ -90,61 +90,77 @@ class InventoryPage extends StatelessWidget {
   Widget _buildBag(BuildContext context, Bags bag, int bagIndex) {
     List<InventoryItem> inventory = bag.inventory.where((i) => i.id != -1 && i.itemInfo != null).toList();
     int usedSlots = inventory.length;
+    FontStyle fontStyle = (usedSlots == bag.size ? FontStyle.italic : FontStyle.normal);
     return CompanionCard(
+      padding: EdgeInsets.all(0.0),
       child: Column(
         children: <Widget>[
-          Row(
-            children: <Widget>[
-              if (bag.itemInfo != null)
-                CompanionItemBox(
-                  item: bag.itemInfo,
-                  hero: '$bagIndex ${bag.id}',
-                  size: 45.0,
-                  includeMargin: false,
-                ),
-              if (bag.itemInfo != null)
-                Expanded(
-                  child: Padding(
-                    padding: const EdgeInsets.all(8.0),
-                    child: Text(
-                      bag.itemInfo.name,
-                      style: Theme.of(context).textTheme.bodyText1.copyWith(
-                        fontWeight: FontWeight.w500
-                      )
+          Container(
+            padding: EdgeInsets.all(8.0),
+            child: Row(
+              children: <Widget>[
+                if (bag.itemInfo != null)
+                  CompanionItemBox(
+                    item: bag.itemInfo,
+                    hero: '$bagIndex ${bag.id}',
+                    size: 45.0,
+                    includeMargin: false,
+                  ),
+                if (bag.itemInfo != null)
+                  Expanded(
+                    child: Padding(
+                      padding: const EdgeInsets.all(8.0),
+                      child: Text(
+                          bag.itemInfo.name,
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyText1
+                              .copyWith(
+                            fontWeight: FontWeight.w500,
+                            fontStyle: fontStyle
+                          )
+                      ),
                     ),
                   ),
-                ),
-              if (bag.itemInfo == null)
-                Spacer(),
-              Text(
-                '$usedSlots/${bag.size}',
-                style: Theme.of(context).textTheme.bodyText1,
-              )
-            ],
+                if (bag.itemInfo == null)
+                  Spacer(),
+                Text(
+                  '$usedSlots/${bag.size}',
+                  style: Theme.of(context).textTheme.bodyText1.copyWith(
+                    fontStyle: fontStyle
+                  ),
+                )
+              ],
+            ),
           ),
+          if (usedSlots > 0)
+            Divider(),
```

Comment:
I've been playing around, and I think you can improve the Divider by setting it's parameters like this:
```
Divider(
    height: 2,
    thickness: 1,
)
```
But feel free to come with other suggestions 😃

Reply:
I've updated this to 2/1 - I fiddled with the numbers, these look good. The intent was just to provide a bit of a differentiation between the heading and the content, so this works really well.